### ISSUE TITLE
Bomb Bums/Key Masters should guarantee a reward after 5 donations and Bomb Bum buff

### ISSUE THREAD

**Issue by ERRRR0R** [UnderscoreKilburn/repentance-issues#1838]
They can get like 10 donations and give nothing so I don't want to waste any time on them even if I have spare keys/bombs, they're too greedy.

They can also be buffed by guaranteeing first reward after 3 donations so you might donate even if you don't have much

And Bomb Bums are really useless, even their item pool is bad (why is there no Pyromaniac btw?). Maybe they can give more rare coins and rarely pickups from super secret rooms like giant chest, wooden chest, black heart, a couple runes, a bunch of hearts etc.

**Comment by Turtier** [comment]
Yeah you spend 5 bombs and get back 2 coins but don't worry if your lucky you will spend 1 bomb and get 2 coins which is still a ripoff :)

The chance for an item either needs to be higher or the other rewards need to actually be close to worth the price of a few bombs.

Key masters however I think are fine, sure you sometimes get ripped off but at the end of the day your getting chests which are so much better than what bomb bums give.

**Comment by batbrain55** [comment]
Instead of coins and stuff, bomb bums could pay out by summoning tinted/fools' gold rocks, which would make them more mechanically similar to the Key Master in terms of you needing bombs both to pay it and to access some of its payouts. Maybe it could also give you rock-themed items, or open the Secret/Super Secret room for you, or give you stuff you find by destroying tile entities, like soul hearts, Dry Baby or even Magic Mushroom.
